test/test_issue1387.m

```diff
@@ -0,0 +1,40 @@
+function test_issue1387
+
```

Comment:
Please add MEM, WALLTIME and DEPENDENCY here

Reply:
OK added. I used `tic;toc` to measure `WALLTIME` and the value returned from ft_preprocessing for `MEM`, doubling both values.